helm_deploy/hmpps-template-typescript/Chart.yaml

```diff
@@ -5,8 +5,8 @@ name: hmpps-template-typescript
 version: 0.2.0
 dependencies:
   - name: generic-service
-    version: 1.5.0
+    version: 2.1.0
     repository: https://ministryofjustice.github.io/hmpps-helm-charts
   - name: generic-prometheus-alerts
-    version: 0.4.1
+    version: 1.1.0
```

Comment:
This major version bump removes a verbose alert for cronjobs. See https://github.com/ministryofjustice/hmpps-helm-charts/pull/67 for more information.